templates/taxi/manufacturer_list.html

```diff
@@ -23,11 +26,19 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url "taxi:manufacturer-update" manufacturer.pk %}" class="btn btn-outline-primary change-link">Update</a>
+                <a href="{% url "taxi:manufacturer-delete" manufacturer.pk %}" class="btn btn-outline-danger change-link">Delete</a>
```

Comment:
Consider using single quotes consistently in the `{% url %}` tag for the 'manufacturer-delete' link to match the style used elsewhere in the template, such as in line 6.